--- a/SCons/Config/gnu	2015-02-11 09:44:40.746132954 +0100
+++ b/SCons/Config/gnu	2015-02-11 09:46:13.410128624 +0100
@@ -71,6 +71,7 @@
 if not defenv['DEBUG']:
 	stub_env.Append(CCFLAGS = ['-Os'])                # optimize for size
 stub_env.Append(CCFLAGS = ['-Wall'])                # all warnings
+stub_env.Append(CCFLAGS = ['-fpermissive'])         # workaround for Name lookup changes
 stub_env.Append(CCFLAGS = ['-xc'])                  # force compile as c
 stub_env.Append(CCFLAGS = ['-fno-strict-aliasing']) # not safe for strict aliasing
 
@@ -91,8 +92,10 @@
 if not defenv['DEBUG']:
 	makensis_env.Append(CCFLAGS = ['-O2'])                  # optimize
 makensis_env.Append(CFLAGS = ['-Wall'])                   # all warnings
+makensis_env.Append(CFLAGS = ['-fpermissive'])            # workaround for Name lookup changes
 makensis_env.Append(CXXFLAGS = ['-Wno-non-virtual-dtor']) # ignore virtual dtor warnings
 makensis_env.Append(CXXFLAGS = ['-Wall'])                 # all warnings
+makensis_env.Append(CXXFLAGS = ['-fpermissive'])          # workaround for Name lookup changes
 
 conf = FlagsConfigure(makensis_env)
 conf.CheckCompileFlag('-m32')                     #
@@ -110,6 +113,7 @@
 if not defenv['DEBUG']:
 	plugin_env.Append(CCFLAGS = ['-Os'])              # optimize for size
 plugin_env.Append(CCFLAGS = ['-Wall'])              # level 3 warnings
+plugin_env.Append(CCFLAGS = ['-fpermissive'])         # workaround for Name lookup changes
 plugin_env.Append(CCFLAGS = ['-fno-strict-aliasing']) # not safe for strict aliasing
 
 if not defenv['DEBUG'] and defenv['STRIP'] and defenv['STRIP_W32']:
@@ -127,6 +131,7 @@
 if not defenv['DEBUG']:
 	cp_util_env.Append(CCFLAGS = ['-O2'])             # optimize
 cp_util_env.Append(CCFLAGS = ['-Wall'])             # all warnings
+cp_util_env.Append(CCFLAGS = ['-fpermissive'])         # workaround for Name lookup changes
 cp_util_env.Append(CCFLAGS = ['-fno-strict-aliasing']) # not safe for strict aliasing
 
 conf = FlagsConfigure(cp_util_env)
